src/__mocks__/openSearchTraceMock.ts

```diff
@@ -0,0 +1,156 @@
+// cSpell:disable
```

Comment:
oh neat I didn't know you could do this I always just add weird words to the list 😆 good to know